### PR TITLE
Create an operations training guide

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

**Comment by yogansun** [comment]
Andre, wondering if a THROTTLED state automatically goes away. Is there any action required by pipeline leads?

Wondering about INFLIGHT/QUEUED status - should those be mentioned in this doco

**Comment by apmasell** [comment]
Updated!

File: ops-guide.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+on any cell header or cell in the tables, the search can be restricted using
+the _Drill Down_ menu item.
+
+For the histograms, clicking and dragging will filter on that range. Histograms
+can provide a lot of useful information about what is going on in an olive. For
+instance:
+
+![Histogram showing changed data](https://raw.githubusercontent.com/oicr-gsi/shesmu/master/ops-histo-orphan.png)
+
+In this histogram there are a nubmer of actions that haven not been generated
```
Comment (asofmosq) [inline ops-guide.md:53]: Non-issue but few typos in this sentence

File: ops-guide.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+- a state
+- a last generation time (a.k.a. `added`)
+- a last checked time (a.k.a. `checked`)
+- a last state transition time (a.k.a. `statusChanged`)
+- an external modification time (a.k.a. `external`; optional)
+
+Every few minutes, Shesmu runs all the olives and they generate all the
+actions. Since most actions are the same every time, the duplicates are thrown
+away. The _last generation time_ is the last time an olive produced this olive.
```
Comment (asofmosq) [inline ops-guide.md:244]: Last time an olive produced this olive??

**Comment by apmasell** [comment]
Updated.

**Pull request merged**